src/pages/K8s.vue

```diff
@@ -169,7 +169,9 @@ export default {
       return os.cpus().length;
     },
     cannotReset() {
-      return ![K8s.State.STARTED, K8s.State.ERROR].includes(this.state);
+      const { STARTED, ERROR, DISABLED } = K8s.State;
```

Comment:
I disagree with this change. If the state is `DISABLED` there's no point enabling the `Reset` button, as it'll have no effect. So this means the code for `cannotReset` can be left as is.

Reply:
The issue here is that `cannotReset()` is also used for determining if the `Enable Kubernetes` toggle is disabled. I added a new computed prop named `cannotToggle` so we can avoid changing the reset button behavior.

Reply:
What if we always allow toggling the `settings.kubernetes.enabled` button? If there's an error trying to bring up kubernetes, the user can still toggle it off and do a restart, and hopefully the VM can come up without a k3s engine.

Reply:
Ah, my mistake ... we always want to allow resetting "kubernetes", as it's actually `Reset System`